.github/workflows/km-ci-workflow.yaml

```diff
@@ -141,56 +142,89 @@ jobs:
       - name: Build and push KM testenv image
         run: make -C tests testenv-image push-testenv-image
 
-      - name: Build payloads and create test images
-        run: |
-          make -C payloads -j pull-buildenv-image
-          make -C payloads -j clean
-          make -C payloads -j all
-          make -C payloads -j testenv-image
-          make -C payloads -j push-testenv-image
-
-      - name: Create payloads runenv and demo-runenv images
-        # Note: this builds both runenv and demo-runenv images
-        run: |
-          make -C payloads -j runenv-image
-          make -C payloads -j push-demo-runenv-image
-
-      # Note: custom Python build needs to happen after python payload build
-      - name: Python.km custom build
-        run: |
-          make -C payloads/python build-modules pack-modules CUSTOM_MODULES="markupsafe wrapt"
-          make -C payloads/python custom CUSTOM_MODULES="markupsafe wrapt"
-
       - name: Build faktory
         run: make -C tools/faktory all
 
       - name: Build shim
         run: |
           make -C cloud/k8s/deploy/shim
-          mkdir /opt/kontain/bin/shim
-          cp -r build/cloud/k8s/deploy/shim/* /opt/kontain/bin/shim
+          mkdir build/opt/kontain/bin/shim
+          cp -r build/cloud/k8s/deploy/shim/* build/opt/kontain/bin/shim
 
       - uses: actions/upload-artifact@v2
         with:
           name: km
           path: |
-            /opt/kontain/bin/km
-            /opt/kontain/bin/krun
-            /opt/kontain/bin/shim
+            build/opt/kontain/bin/km
+            build/opt/kontain/bin/kontain-gcc
+            build/opt/kontain/bin/krun
+            build/opt/kontain/bin/shim
           retention-days: 7
 
+      # passign hello_test.kmd that is required by testenv of payloads - it already has correct RPATH.
+      # passing hello_test.c and libhelper.a
+      # because we will need to rebuid hello_test.kmd with correct RPATH for runtime environments
```

Comment:
This line and two above sounds contradictory, or at least confusing

Reply:
Not sure what is confusing - using already existing .kmd for testing and will re-build for runner/demo-runenv